pkg/operator/condition/condition.go

```diff
@@ -0,0 +1,56 @@
+package condition
+
+const (
```

Comment:
Usually we introduce a condition type (string alias) and type the constant that way.

Reply:
@sttts right, but it will require openshift/api type and change in the operator status? or type casting on usage with sucks.